### PR TITLE
[UI] Fixed build/dev for linux

### PR DESCRIPTION
**Pull request by dzannotti**
By updating flow and node-sass, because both versions used were known to cause issues in linux envs

### CODE REVIEW

File: digdag-ui/model.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ export type Task = {
 export type Attempt = {
   id: number;
   project: IdName;
-  workflow: NameOptionalId;
+  workflow: NameId;
```
Comment (danielnorberg) [inline digdag-ui/model.js]: The `Attempt` type should probably match the underlying rest api type.

https://github.com/treasure-data/digdag/blob/master/digdag-client/src/main/java/io/digdag/client/api/RestSessionAttempt.java#L23

File: digdag-ui/model.js [hunk 2]
```diff
@@ -27,9 +27,9 @@ export type IdName = {
   name: string;
 }
 
-export type NameOptionalId = {
+export type NameId = {
   name: string;
-  id: ?number;
+  id: number;
```
Comment (danielnorberg) [inline digdag-ui/model.js]: https://github.com/treasure-data/digdag/blob/master/digdag-client/src/main/java/io/digdag/client/api/NameOptionalId.java

Comment (dzannotti) [inline digdag-ui/model.js]: True and agreed, however the JS ui handles the id as being always present (see https://github.com/treasure-data/digdag/blob/master/digdag-ui/console.jsx#L299) Should the correct behaviour be checking against the existance of that id js side?

Comment (danielnorberg) [inline digdag-ui/model.js]: Indeed. Perhaps it should.

**Pull request merged**